pom.xml

```diff
@@ -125,7 +125,7 @@
     <!--<dynamodb-local.repository.url>https://s3.us-west-2.amazonaws.com/dynamodb-local/release</dynamodb-local.repository.url>-->
 
     <!-- Version properties -->
-    <antlr.version>4.9.2</antlr.version>
+    <antlr.version>4.8</antlr.version>
```

Comment:
nit: Iceberg uses 4.7.1

Reply:
yeah, i wanted to keep to spark3 rather than iceberg. I suspect iceberg probably wants to bump also.